static/src/javascripts/projects/common/modules/experiments/tests/membership-message-usa.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/modules/identity/api',
+    'common/modules/ui/message',
+    'text!common/views/membership-message.html',
+    'common/views/svgs'
+], function (
+    config,
+    detect,
+    storage,
+    template,
+    idApi,
+    Message,
+    messageTemplate,
+    svgs
+) {
+
+    return function () {
+
+        this.id = 'MembershipMessageUsa';
+        this.start = '2015-08-05';
+        this.expiry = '2015-09-21';
+        this.author = 'David Rapson';
+        this.description = 'Test if loyal visitors in the US edition are encouraged to join Membership as a Supporter';
+        this.audience = 1;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Loyal visitors will be interested in becoming a Supporter';
+        this.audienceCriteria = 'US edition visitors who have seen at least 10 pages, message shown only on article pages';
+        this.dataLinkNames = 'membership message, hide, read more';
+        this.idealOutcome = 'Users will sign up as a Supporter';
+
+        this.canRun = function () {
+            /**
+             * - Exclude adblock users to avoid conflicts with similar adblock Supporter message
+             * - Exclude mobile/small-screen devices
+             * - Only show for US edition
+             * - Only show on Article pages
+             * - Only show to visitors who have viewed at least 10 pages.
+             */
+            var alreadyVisited = storage.local.get('alreadyVisited') || 0;
```

Comment:
@OliverJAsh Yeah, this is a very valid point. I've been relying on it in the original UK message version too (see: [membership-message.js](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/membership-message.js)) which is far from ideal. Would be great to move this further update the chain. In fact ideally I'd prefer to use a count of **articles** used rather than the fairly broad metric of pageviews anyway.